packages/tsconfig-reference/copy/en/options/rewriteRelativeImportExtensions.md

```diff
@@ -1,6 +1,6 @@
 ---
 display: "rewriteRelativeImportExtensions"
-oneline: "Does something"
+oneline: "Rewrite TypeScript extensions of relative import paths to their JavaScript equivalent."
```

Comment:
Normally this is a copy of https://github.com/microsoft/TypeScript/blob/34ea32f8ce2771fb3b063f66ec0bc7b0c017d9e4/src/compiler/commandLineParser.ts#L1207 which is actually just the string below.